notebook/static/notebook/js/menubar.js

```diff
@@ -128,6 +128,142 @@ define([
 
         this._new_window(url);
     };
+    
+    MenuBar.prototype._nbconvert_upload_conf = function (download) {
+      
+      var body =  $('<div>');
+      var notebook_path = utils.encode_uri_components(this.notebook.notebook_path);
+
+      var create_json = function(notebook, config){
+        var json_to_pass = {
+          notebook: notebook.toJSON(),
+          config: config
+        };
+        return json_to_pass;
+      };
+
+      
+      var form  = $('<form>');
+      var fileinput = $('<input>')
+        .attr('type', 'file')
+        .attr('tabindex', '1');
+
+      var fileformat = $('<select>');
+      
+      var export_opts = {
+        markdown: {
+          display_text:'markdown',
+          exporter_name:'markdown' 
+          },
+        html: {
+          display_text:'html',
+          exporter_name:'html' 
+          },
+        rst: {
+          display_text:'rst',
+          exporter_name:'rst' 
+          },
+        pdf: {
+          display_text:'PDF',
+          exporter_name:'pdf' 
+          },
+        latex: {
+          display_text:'LaTeX',
+          exporter_name:'latex' 
+          },
+        script: {
+          display_text:'script',
+          exporter_name:'script' 
+          }
+      };
+
+      for(var x in export_opts){
+        fileformat.append(
+          $('<option/>') 
+            .attr('value', export_opts[x]['exporter_name'])
+            .text(export_opts[x]['display_text'])
+        );
+      };
+      
+      form.append(fileformat);
+      form.append(fileinput);
+      body.append(form);
+      
+      var that = this;
+
+      var handle_nbconvert_post = function(){
```

Comment:
It technically is receiving the file if it has been uploaded when clicking the "Download" button. 

But, ok, it's changed to `trigger_nbconvert_post`.